styles/styles.css

```
@@ -457,10 +655,24 @@ body {
   line-height: 21px;
   width:292px;
   height:50px;
-  border:1px solid #E5E5E5;
+  border:2px solid #E5E5E5;
   border-radius: 4px;
   margin-right: 70px;
-   }
+  margin-left: 200px;
+}
+.select-control:hover{
+  border:2px solid #68A2CA;
+}
+.select-control:focus{
+  border:2px solid #68A2CA;
+}
```

вот такие вещи лучше записывать так 
```suggestion
.select-control:hover, select-control:focus{
  border:2px solid #68A2CA;
}
```

что бы дублирование кода снизить